<launch>
    <!-- Launch arguments -->
    <arg name="cfg_yaml" 		default="$(find face_shooter)/config/track.yaml" />
    <arg name="img_topic" 		default="/ardrone/front/image_raw" />
    <arg name="centroid_topic" 	default="/face_centroid" />
    <arg name="display_original_image"       default="false" />
    <arg name="display_tracking_image"       default="true" />

    <!-- load the detection parameters   -->
    <rosparam file="$(arg cfg_yaml)" command="load"/>

    <!-- Start the node to track a face in an video stream   -->
    <node pkg="face_shooter" type="face_tracker_imgmsg.py" name="face_tracker_imgmsg" output="screen" >
        <param name="/display_original_image"       value="$(arg display_original_image)" />
        <param name="/display_tracking_image"       value="$(arg display_tracking_image)" />
        <param name="/subscribed_image_topic"    	value="$(arg img_topic)" />
        <param name="/publishing_centroid_topic"    value="$(arg centroid_topic)" />
        <param name="/haar_face_file"    			value="$(find face_shooter)/data/face.xml" />
    </node>

</launch>

